router/docs.js

```diff
@@ -10,84 +10,84 @@ const docsRoutes = require("./routes/docs");
 const uploader = multer({ dest: "uploads/" });
 const router = express.Router();
 router.use(requireLoggedIn);
-router.use("/docs", docsRoutes);
+
 
 // Render all of a user's documents
-router.get("/", function(req, res) {
+router.get("/home", function(req, res) {
```

Comment:
This would be at `/docs/home` (Which by the way, your app is not the docs app, you shouldn't be calling things docs.) Are you sure you want both `/home` (Defined in app.js) and this new `/docs/home`?